hello-app-iap/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hello-app-iap
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: hello-app-iap
+    spec:
+      containers:
+      - name: app
+        image: gcr.io/google-samples/hello-app-iap:1.0.0
+        ports:
+          - containerPort: 8080
+        readinessProbe:
+          httpGet:
+            host: "{{ENDPOINT}}"
```

Comment:
do you need `host` in this case? also how does the probe authenticate to the `/_gcp_iap/identity` to get an `http 200 OK`?

Reply:
It doesn’t. This path only exists when IAP is enabled on the backend service. This probe ensures that traffic never goes to the pod without IAP first being enabled.